### DISCUSSION TITLE
Abandoned Environment

### DISCUSSION THREAD

**Discussion by tqpcharlie** [community/community#23997] [Other Feature Feedback, Questions, & Ideas]
<p>So I have the github pages environment set up in my repo, but all of a sudden it says that the github-pages environment is abandoned. What does this mean. <a href="https://github.com/Zachinquarantine/gr-eight-homework-checker" rel="noopener nofollow ugc">Link to my repository</a>.</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>You page seems to be working - where exactly are you seeing this error message?</p>

**Comment by tqpcharlie** [comment, marked as answer]
<p>Hello, it resolved when I removed github pages and readded it. Thanks anyway!</p>
